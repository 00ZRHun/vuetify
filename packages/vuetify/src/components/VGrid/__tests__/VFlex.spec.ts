// Components
import VFlex from '../VFlex'

// Utilities
import {
  mount,
  MountOptions,
  Wrapper
} from '@vue/test-utils'

describe('VFlex.ts', () => {
  type Instance = InstanceType<typeof VFlex>
  let mountFunction: (options?: MountOptions<Instance>) => Wrapper<Instance>

  beforeEach(() => {
    mountFunction = (options = {}) => mount(VFlex, options)
  })

  it('should render', () => {
    const wrapper = mountFunction()

    expect(wrapper.html()).toMatchSnapshot()
  })
})
