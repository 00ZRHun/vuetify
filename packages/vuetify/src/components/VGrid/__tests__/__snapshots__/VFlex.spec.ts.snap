// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VFlex.ts should render 1`] = `

<div class="flex">
</div>

`;
